### ISSUE TITLE
Control over data types and sizes

### ISSUE THREAD

**Issue by EnriqueL8** [IBM-Swift/Swift-Kuery-ORM#20]
A way of specifying the data types used in the database for the model.

**Comment by xeokeri** [comment]
I agree with this. I have an enum that subclasses string, but it should be a `varchar` or `char`, instead it creates `text`, but then has issues with then enum class when doing additional QueryParam filtering. As it doesn't see the `Enum : String`, but as an default and converting to `String`. Having more fine grain control over it would also assist with the MySQL bind issue I'm having locally trying to find a work-around on this in the meantime.
